wagtail/wagtailcore/__init__.py

```diff
@@ -3,6 +3,8 @@
 # Imported for historical reasons
 from wagtail import __semver__, __version__  # noqa
 
+default_app_config = 'wagtail.wagtailcore.apps.WagtailCoreAppConfig'
```

Comment:
Looks like this was removed in https://github.com/wagtail/wagtail/commit/384d95d2b5937b3db63ee0d82892c1d18024f96a (I assume this was an accident)